src/main/java/seedu/cafectrl/data/Pantry.java

```diff
@@ -147,19 +151,29 @@ public void calculateDishAvailability(Menu menu) {
      *
      * @param dish The dish being ordered.
      */
-    public int calculateMaxDishes(Dish dish, Menu menu) {
+    public int calculateMaxDishes(Dish dish, Menu menu, Order order) {
         int maxNumofDish = Integer.MAX_VALUE;
         ArrayList<Ingredient> dishIngredients = retrieveIngredientsForDish(dish.getName(), menu);
+        boolean restockHeaderDisplayed = false;

```

Comment:
perhaps a more appropriate name would be "isRestockHeaderDisplayed"?

Reply:
Thank you for reminding me!